go/libkb/constants.go

```diff
@@ -317,7 +317,7 @@ const (
 	LinkTypeDeleteRoot       LinkType = "team.delete_root"
 	LinkTypeDeleteSubteam    LinkType = "team.delete_subteam"
 	LinkTypeDeleteUpPointer  LinkType = "team.delete_up_pointer"
-	LinkTypeLegacyTLFUpgrade LinkType = "team.legacy_tlf_upgrade"
+	LinkTypeKBFSSettings     LinkType = "team.kbfs"
```

Comment:
Unfortunately changing this string will mean that client versions prior to this PR will **reject** team sigchains that include this link. Due to this check https://github.com/keybase/client/blob/master/go/teams/loader_types.go#L133

If this were a new link type this would not come up. It's happening because this link type was in "IsSupportedTeamType".

I could have this wrong, best to test either way.

Reply:
good point, i'll retire the old sequence number.